CSharpCodeGenerator/CSharpCodeGenerator/Program.cs

```diff
@@ -72,6 +72,11 @@ private static void Run(Options options)
                     .OrderByDescending(t => t.QualifiedName.Count(c => c == '.'))
                     .ToList();
 
+                foreach (var w in types.SelectMany(t => t.Warnings))
+                {
+                    Console.WriteLine(w);
```

Comment:
Can it be made clearer that these are warnings? Either with color or a prefix?

Reply:
Yeah, I considered including Serilog, I guess it won't add too many more lines to the changelist anyway...